web/styles/zulip.css

```diff
@@ -2064,6 +2069,13 @@ td.pointer {
         .private_message_header_name {
             overflow: hidden;
             text-overflow: ellipsis;
+            display: flex;
+            align-items: center;
+
+            i {
```

Comment:
And here. Also, can more be done with this element as a flex item? Is margin really the best way to go here?

Reply:
I think using the `gap` property would be better here, rather than `margin`.